python/ray/tests/test_task_metrics.py

```diff
@@ -48,33 +51,34 @@ def tasks_by_state(info) -> dict:
         return {}
 
 
-# TODO(ekl) in all these tests, we use run_string_as_driver_nonblocking to work around
-# stats reporting issues if Ray is repeatedly restarted in unit tests.
-def test_task_basic(shutdown_only):
-    info = ray.init(num_cpus=2, **METRIC_CONFIG)
+@pytest.mark.skipif(sys.platform == "win32", reason="Flaky on Windows.")
+def test_metrics_export_now(shutdown_only):
```

Comment:
Interestingly, if I run this test after running `test_basic`, it has corrupted metrics. But if I schedule any task, they are cleaned up for some reasons...